Sources/AVPlayer+Android.swift

```diff
@@ -10,7 +10,9 @@ import JNI
 
 public class AVPlayer: JNIObject {
     public var onLoaded: ((Error?) -> Void)?
+    public var onVideoReady: (() -> Void)?
     public var onVideoEnded: (() -> Void)?
+    public var onVideoStalled: (() -> Void)?
```

Comment:
nit:
I'd prefer the callback to be named similar to the actual event so `onVideoBuffering`

Reply:
Ah right, the reason why I named it `onVideoStalled` was to resemble [AVPlayerItemPlaybackStalled](https://developer.apple.com/documentation/foundation/nsnotification/name/1387661-avplayeritemplaybackstalled). But now that I look at the code again there is no reason to do that. Will change it 👍